src/main/java/io/chrislowe/discordle/Main.java

```diff
@@ -112,23 +112,33 @@ private static void registerCommands(GatewayDiscordClient gateway) {
 
                 if (response != null) {
                     yield event.reply(response);
-                } else {
-                    yield event.deferReply().then(createGameBoardFollowup(event));
+                }
+                else {
```

Comment:
This is very minor, but this project is following standard Oracle styling guidelines, which have an `else` statement on the same line as the previous bracket, like this:
`} else {`

Reply:
Done!